agents-api/migrations/migrate_1733755642_transition_indices.py

```diff
@@ -0,0 +1,26 @@
+# /usr/bin/env python3
+
+MIGRATION_ID = "transition_indices"
+CREATED_AT = 1733755642.881131
+
+
+create_transition_indices = dict(
+    up="""
+    ::index create executions:execution_id_status_idx { status, execution_id }
+    ::index create transitions:execution_id_type_idx { type, execution_id }
```

Comment:
this will help more instead:
`::index create transitions:execution_id_type_created_at_idx { execution_id, type, created_at }`

and then explicitly use it in the query `transitions:execution_id_type_created_at_idx`